airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/ConnectionTransformationTab/components/DbtCloudTransformationsCard.tsx

```diff
@@ -17,9 +17,9 @@ import { useCurrentWorkspace } from "hooks/services/useWorkspace";
 import { DbtCloudJob, isSameJob, useDbtIntegration, useAvailableDbtJobs } from "packages/cloud/services/dbtCloud";
 import { RoutePaths } from "pages/routePaths";
 
-import dbtLogo from "./dbt-bit_tm.svg";
+import dbtLogo from "../dbt-bit_tm.svg";
+import octaviaWorker from "../octavia-worker.png";
```

Comment:
Why are we not using an SVG for the octaviaWorker image?

Curious why these are being imported this way and not as components?